web/test/pages/profile/profile-owning.test.tsx

```diff
@@ -477,150 +478,181 @@ describe("profile - owning existing business", () => {
     });
   });
 
-  describe("profile opportunities alert", () => {
+  describe("profile opportunities alert for Oscar", () => {
```

Comment:
this is unnecessary because the test is `profile-owning`

Reply:
relabeled